src/main/java/org/cbehrens/spotifycodingchallenge/album/AlbumController.java

```diff
@@ -0,0 +1,46 @@
+package org.cbehrens.spotifycodingchallenge.album;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/album")
+public class AlbumController {
+
+    private final AlbumApplicationService albumApplicationService;
+
+    @Autowired
+    public AlbumController(AlbumApplicationService albumApplicationService) {
+        this.albumApplicationService = albumApplicationService;
+    }
+
+    @GetMapping(value = "/{id}")
```

Comment:
```suggestion
    @GetMapping("/{id}")
```
this goes for all the endpoints here with a subpath